asset/css/doc.css

```diff
@@ -203,9 +203,6 @@ div.odoc .comment-delim {
 }
 
 .navmap-tag {
-  display:flex;
-  align-items: center;
-  justify-content: center;
```

Comment:
is this an intentional change? looks unrelated to dark mode and should probably not be in this patch

Reply:
I can't remember why I removed this, but I added it back now and everything is still good. Probably I was testing how some styles work and forgot to add it back